plan/plans.go

```diff
@@ -46,6 +46,16 @@ type CheckTable struct {
 	Tables []*ast.TableName
 }
 
+// CheckIndex is used for checking index data, output the index values that handle within begin and end.
+type CheckIndex struct {
+	basePlan
+
+	Table     *ast.TableName
+	IndexName string
+	Begin     int64
```

Comment:
So we could only check index with integer column?

Reply:
This is the handle range, not the index value range.